db/migrate/20150110084027_change_team_slug_to_citext.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeTeamSlugToCitext < ActiveRecord::Migration
+  def change
+    change_column :teams, :slug, :citext, null: false, index: true , unique: true
```

Comment:
The schema.rb not changing is okay, as long as the index does get added. I ran this same migration on my local and checked whether the index is present via `psql`, but it doesn't show. The only index present is `"teams_pkey" PRIMARY KEY, btree (id)`